README.md

```diff
@@ -41,7 +48,7 @@ git clone https://github.com/zivid/cpp-extra-samples
 [comment]: <> (Choose a sample solution and configure it with CMake.)
 [comment]: <> (Launch Visual Studio, open, build, and run the sample solution.)
 
-Choose a sample solution, configure it with CMake, open it in Visual Studio, build it, run it. If you are uncertain about doing this, check out our tutorials for configuring [**Zivid**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/61472793/Configure+Zivid+Extra+Samples+with+CMake+and+build+them+using+Visual+Studio+in+Windows) and  [**Zivid & PCL**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/44204066/Configure+C+Zivid+Point+Cloud+Library+PCL+Samples+with+CMake+and+build+them+using+Visual+Studio+in+Windows) samples with CMake and building them using Visual Studio in Windows.
+Choose a sample solution, configure it with CMake, open it in Visual Studio, build it, run it. If you are uncertain about doing this, check out our tutorials for configuring [**Zivid**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/61472793/Configure+Zivid+Extra+Samples+with+CMake+and+build+them+using+Visual+Studio+in+Windows), [**Zivid & PCL**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/44204066/Configure+C+Zivid+Point+Cloud+Library+PCL+Samples+with+CMake+and+build+them+using+Visual+Studio+in+Windows), and [**Downsample**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/76480513/Configure+C+Downsample+Sample+with+CMake+and+build+it+using+Visual+Studio+in+Windows) samples with CMake and building them using Visual Studio in Windows.
```

Comment:
The Downsample link tells me the page is restricted, I assume you plan on publishing it after the PR is merged?

Reply:
Yes, that is correct.